### PR TITLE
T5-290

### PR DESCRIPTION
**Pull request by naeluh**
https://thirdandgrove.atlassian.net/browse/T5-290

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -19,46 +20,52 @@ const Index = ({ data }) => {
...
+  ];
+  // If include array isset reorder array and filter
+  const reorderArray = (arr, order) => {
+    const newArr = [];
 
-  // silly holiday treats
-  // const today = new Date();
-  // const isHalloween = today.getMonth() === 9 && today.getDate() === 31;
-  // const isChristmas = today.getMonth() === 11 && today.getDate() === 25;
+    for (let i = 0; i < arr.length; i++) {
```
Comment (jds-tag) [inline src/pages/index.js:36]: Any way to avoid nested for loops? Any reason not to use `forEach`?

**Review by jds-tag — COMMENTED** [review]
One nit

File: src/pages/index.js (hunk 1 above)
Comment (naeluh) [reply src/pages/index.js]: I think I have to have the nested loops since its comparing 2 arrays, but I added `forEach` thanks!

**Review by jds-tag — APPROVED** [review]
LGTM!

**Pull request merged**